quoridorui/src/main/java/com/harush/zitoon/quoridor/ui/controller/MainMenuController.java

```diff
@@ -35,7 +37,7 @@ private void onPlayBtnPress(ActionEvent event) {
      */
     @FXML 
     private void onHelpBtnPress(ActionEvent event) {
-    	Stage stage = (Stage) exitButton.getScene().getWindow();
+    	Stage stage = (Stage) button.getScene().getWindow();
```

Comment:
see, like over here, when you press the help button, you refrencing a local variable that calls exit button while its should be just button. 
its just left overs from the old fork